uaclient/tests/test_patch_status_json.py

```diff
@@ -0,0 +1,134 @@
+import logging
+import pytest
+
+import json
+
+from lib.patch_status_json import patch_status_json_schema_0_1
+
+UNATTACHED_UNPATCHED_STATUS = {
+    "_doc": "Content provided in json response is currently considered ...",
+    "attached": False,
+    "expires": "n/a",
+    "techSupportLevel": "n/a",
+    "notices": [],
+    "services": [
+        {
+            "name": "cc-eal",
+            "available": "yes",
+            "description": "Common Criteria EAL2 Provisioning Packages",
+        }
+    ],
+    "origin": None,
+    "configStatusDetails": "No Ubuntu Advantage operations are running",
+    "configStatus": "inactive",
+}
+
+UNATTACHED_PATCHED_STATUS = {
+    "execution_details": "No Ubuntu Advantage operations are running",
+    "_schema_version": "0.1",
+    "origin": None,
+    "account": {"name": "", "id": ""},
+    "expires": "n/a",
```

Comment:
Probably leaving these minor discrepancies unaddressed because we are only performing the minimal set of key changes to support non-root user runs of `ua status`. It allows us to keep this script minimal until we have official schema compatibility support for status.json where we can construct each "supported" schema version of status.json via a true patching up/down set of operations.